### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cmungall**

### CODE REVIEW

**Comment by oneilsh** [comment]
Woohoo github - since I haven't as yet done large team projects with github, please excuse my ettiquitte and feel free to correct whenever :D I added you as an admin to the repo. 

Anyway, looking at this PR, the current dev server runs on 3434 right?

**Comment by cmungall** [comment]
good catch! I fixed. btw you can comment on individual lines directly via the web UI just click on a line

image

File: README.md [hunk 1]
```diff
@@ -2,6 +2,44 @@
...
+
+### Register
+
+Go to https://chat.openai.com/
+
+If you have plugin access, you'll see an option
+
+Select the plugin model from the top drop down, then select “Plugins”, “Plugin Store”, and finally “Install an unverified plugin” or “Develop your own plugin”.
+
+Enter `localhost:3535`
```
Comment (oneilsh) [inline README.md]: Here too (test test, is this thing on)

**Comment by oneilsh** [comment]
Now that is some kind of sorcery. Thank you :) Since I've never done it before I added a comment as a review, I think I did it right?

**Comment by oneilsh** [comment]
@cmungall silly github question: is it standard practice to delete such patch branches after merge? I'm guessing so in most cases?

**Pull request merged**